### DISCUSSION TITLE
Is it possible to call an endpoint from serverpod_cli?

### DISCUSSION THREAD

**Discussion by y3l1n4ung** [serverpod/serverpod#1455] [Q&A] [answered]
I want to create a new command for user creation, so I need to call an endpoint from serverpod_cli. Is that possible?

**Comment by vlidholt** [comment, marked as answer]
This is currently not possible, other than (very few) built in features that are using the Serverpod Insights API.

The best way is to create your own endpoint for this, and perhaps an Admin interface (at least for now).
